app/src/main/java/com/monkeyteam/chimpagne/ui/components/eventview/OrganiserView.kt

```diff
@@ -80,7 +80,7 @@ fun OrganiserView(
 
         Box(
             modifier =
-                Modifier.testTag("share")
+                Modifier.testTag("shareevent")
```

Comment:
please use either _ or spaces between words for test tags

Reply:
You helped me fix a test with this comment thank you !